templates/flavors/machinepool-windows/machine-pool-deployment-windows.yaml

```diff
@@ -59,6 +59,7 @@ spec:
       criSocket: npipe:////./pipe/containerd-containerd
       kubeletExtraArgs:
         cloud-provider: external
+        register-with-taints: node.cloudprovider.kubernetes.io/uninitialized=true:NoSchedule
```

Comment:
We shouldn't need to add this here since the kubelet always adds this when `cloud-provider=external`:

https://github.com/kubernetes/kubernetes/blob/v1.34.2/pkg/kubelet/kubelet_node_status.go#L337-L345